xarray/core/resample.py

```diff
@@ -58,6 +60,60 @@ def _flox_reduce(
         result = result.rename({RESAMPLE_DIM: self._group_dim})
         return result
 
+    def shuffle(self, chunks: T_Chunks = None):
+        """
+        Sort or "shuffle" the underlying object.
+
+        "Shuffle" means the object is sorted so that all group members occur sequentially,
+        in the same chunk. Multiple groups may occur in the same chunk.
+        This method is particularly useful for chunked arrays (e.g. dask, cubed).
+        particularly when you need to map a function that requires all members of a group
+        to be present in a single chunk. For chunked array types, the order of appearance
+        is not guaranteed, but will depend on the input chunking.
+
+        .. warning::
+
+           With resampling it is a lot better to use ``.chunk`` instead of ``.shuffle``,
+           since one can only resample a sorted time coordinate.
```

Comment:
I'm sure I'm being slow here — but doesn't `.shuffle` claim to sort the array? 

Or this means "when resampling with a dimension _other than_ the chunking dimension"?

Reply:
Yes this is confusing. 

Resampling only works with sorted axes, so shuffling is basically identical to rechunking. Maybe I should just delete this line?